.travis.yml

```diff
@@ -1,10 +1,17 @@
-sudo: false
 language: node_js
+sudo: false
+
 node_js:
     - "0.10"
     - "0.12"
-    - "4"
-    - "5"
     - "iojs"
     - "iojs-v1"
     - "iojs-v2"
+    - "4"
+    - "5"
+
+cache:
+  directories:
+    - node_modules
+after_script:
+  - npm run coverage
```

Comment:
Hey we have an "approved travis.ci" template [here](https://github.com/continuationlabs/cl.ci/blob/master/js.travis.yml) so let's stick with that.

While I don't really have an issue with coveralls, we already get code coverage from lab directly so we don't really need that here.